doc/nrf/gs_assistant.rst

```diff
@@ -54,11 +54,14 @@ Once you have installed the Toolchain manager, open it in nRF Connect for Deskto
 
 Click :guilabel:`Settings` in the navigation bar to specify where you want to install the |NCS|.
 Then, in :guilabel:`SDK Environments`, click the :guilabel:`Install` button next to the |NCS| version that you want to install.
-The |NCS| version of your choice is installed on your machine, and the :guilabel:`Install` button changes to :guilabel:`Open IDE`.
+The |NCS| version of your choice is installed on your machine.
 
-There are two ways you can build an application:
+There are several ways you can build an application:
+
+1. To build with |VSC|, click on the :guilabel:`Open VS Code` button.
+
+#. To build with |SES|, click on the :guilabel:`Open Segger Embedded Studio` button.
 
-1. To :ref:`build with SES <gs_programming_ses>`, click on the :guilabel:`Open IDE` button.
 #. To build on the command line, use the following steps:
```

Comment:
```suggestion
* To build with |VSC|, click on the :guilabel:`Open VS Code` button.
* To build with |SES|, click on the :guilabel:`Open Segger Embedded Studio` button.
* To build on the command line, use the following steps:
```
These are not ordered steps, but exclusive options, no? I would see this as bullets...

Reply:
Updated with bullets.